docusaurus/docs/dev-docs/intro.md

```diff
@@ -7,18 +7,27 @@ sidebar_position: 1
 
 # Welcome to the Strapi Developer Docs!
 
-This documentation contains all technical documentation related to the setup, deployment, update, and customization of your Strapi application.
+<SubtleCallout title="Developer Docs, User Guide, and Strapi Cloud documentation" emoji="📍">
 
-:::strapi Can't wait to start using Strapi?
-You can directly head to the [Quick Start](./quick-start.md)! <br/> If demos are more your thing, we have a [video demo](https://youtu.be/zd0_S_FPzKg), or you can request a [live demo](https://strapi.io/demo)!
-:::
+The documentation for Strapi contains 3 main sections, accessible from the top navigation bar:
+
+- 🧑‍🏫 The **[User Guide](/user-docs/intro)** is all about using Strapi's admin panel.
+- ☁️ The **[Strapi Cloud](/cloud/intro)** documentation is about deploying your Strapi application to Strapi Cloud and managing your Strapi Cloud projects and settings.
+- 🧑‍💻 The **Developer Docs** that you're currently reading contain all the technical information related to the setup, advanced usage, customization, and update of your Strapi v5 application.
```

Comment:
I would suggest using the same order as in the top navigation bar, to make it even more coherent 😊